src/platform/unittest/TlsTest.cpp

```diff
@@ -55,14 +58,23 @@ struct TlsTest : public ::testing::TestWithParam<bool>
 
     static void SetUpTestSuite()
     {
-        SelfSignedCertParams = CxPlatPlatGetSelfSignedCert(CXPLAT_SELF_SIGN_CERT_USER);
+        SelfSignedCertParams = (QUIC_CREDENTIAL_CONFIG*)CxPlatPlatGetSelfSignedCert(CXPLAT_SELF_SIGN_CERT_USER, FALSE);
```

Comment:
This cast wasn't needed before. Why now?

Reply:
Because I'm removing the `const` keyword so I can update the flags field.

Reply:
Ok.